notebooks/200-omics_storage.ipynb

```diff
@@ -382,6 +511,7 @@
   },
   {
    "cell_type": "markdown",
+   "id": "42a642e8",
    "metadata": {},
    "source": [
     "You can check the status here either using `omics.get_reference_import_job` or by navigating to the [Console](https://us-east-1.console.aws.amazon.com/omics/home?region=us-east-1#/reference/referenceStoreDetail)."
```

Comment:
Should update this Console link to be region agnostic:
https://console.aws.amazon.com/omics/home#/reference/referenceStoreDetail